Firestore/Example/Tests/Local/FSTRemoteDocumentCacheTests.m

```diff
@@ -114,6 +114,7 @@ - (void)testDocumentsMatchingQuery {
   FSTDocumentDictionary *results = [self.remoteDocumentCache documentsMatchingQuery:query];
   NSArray *expected =
       @[ FSTTestDoc(@"b/1", kVersion, _kDocData, NO), FSTTestDoc(@"b/2", kVersion, _kDocData, NO) ];
+  XCTAssertEqual([results count], [expected count]);
```

Comment:
Can you also port the comment from JS (which is a little more descriptive than we we have here).

    // TODO(mikelehen): This just verifies that we do a prefix scan against the
    // query path. We'll need more tests once we add index support.

Reply:
Done.